### ISSUE TITLE
[Bug] Unable to create dir in private data dir

### ISSUE THREAD

**Issue by ruattd** [AndroidIDEOfficial/AndroidIDE#766]
### What happened?

When I open IDE, I received a message that to install SDK tools, but it reported an error after confirming.

Stacktrace:
```
com.itsaky.androidide.utils.BootstrapInstaller$InstallationException: Unable to create directory: /data/data/com.itsaky.androidide/files/usr/bin
	at io.github.rosemoe.sora.widget.CodeEditor$$ExternalSyntheticLambda2.run(R8$$SyntheticClass:469)
 Caused by: java.util.concurrent.CompletionException: com.itsaky.androidide.utils.BootstrapInstaller$InstallationException: Unable to create directory: /data/data/com.itsaky.androidide/files/usr/bin
	at io.github.rosemoe.sora.widget.CodeEditor$$ExternalSyntheticLambda2.run(R8$$SyntheticClass:472)
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1663)
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1655)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:285)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1155)
	at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1993)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1941)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```

### What's the expected behavior?

Succeed to install SDK tools.

### What version of AndroidIDE you're using?

v2.1.4

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by deepkrg17** [comment]
Most probably fixed in ff20e65e571efe173ff487fec3048c9e810085bf.
Try the apk from GitHub actions or wait for the next release.

**Comment by itsaky** [comment]
#1450.
